js/libs/form.js

```diff
@@ -50,6 +50,10 @@ Fliplet.Widget.instance('form-builder', function(data) {
             } else {
               field.value = moment(entry.data[field.name]).format('DD/MM/YYYY');
             }
+          } else if (field._type === "flCheckbox" && !entry.data[field.name]) {
```

Comment:
the last condition should be replaced with `!Array.isArray(entry.data[field.name])`